src/main/java/network/brightspots/rcv/GuiApplication.java

```diff
@@ -27,7 +27,7 @@
 import javafx.stage.Stage;
 
 @SuppressWarnings("WeakerAccess")
```

Comment:
You might be able to get rid of this suppression now too.

Reply:
I'll try and tackle all these suppressions in a later change.

Reply:
Thanks for removing this suppression. I think you may have gone overboard removing the other ones though. They're in place to keep IntelliJ from complaining in its analysis.